glue_lab/glue_session.py

```diff
@@ -97,73 +110,99 @@ def render_viewer(self) -> None:
+                                viewer_data=data,
+                                viewer_state=state,
+                            )
+                        }
                     else:
-                        data = None
-                    if view_type == "glue.viewers.scatter.qt.data_viewer.ScatterViewer":
-                        output.clear_output()
-                        try:
-                            with output:
-                                scatter = self.app.scatter2d(data=data)
-                                for key, value in state.items():
-                                    try:
-                                        setattr(scatter.state, key, value)
-                                    except Exception:
-                                        pass
-                        except Exception as e:
-                            output.clear_output()
-                            with output:
-                                print(e)
-
-                    elif view_type == "glue.viewers.image.qt.data_viewer.ImageViewer":
-                        output.clear_output()
-                        try:
-                            with output:
-                                self.app.imshow(data=data)
-                        except Exception as e:
-                            output.clear_output()
-                            with output:
-                                print(e)
-                    elif (
-                        view_type
-                        == "glue.viewers.histogram.qt.data_viewer.HistogramViewer"
-                    ):
-                        output.clear_output()
-                        try:
-                            with output:
-                                hist = self.app.histogram1d(data=data)
-                                for key, value in state.items():
-                                    try:
-                                        setattr(hist.state, key, value)
-                                    except Exception:
-                                        pass
-                        except Exception as e:
-                            output.clear_output()
-                            with output:
-                                print(e)
-                    elif view_type == "glue.viewers.table.qt.data_viewer.TableViewer":
-                        output.clear_output()
-                        try:
-                            with output:
-                                table = self.app.table(data=data)
-                                for key, value in state.items():
-                                    try:
-                                        setattr(table.state, key, value)
-                                    except Exception:
-                                        pass
-                        except Exception as e:
-                            output.clear_output()
-                            with output:
```

Comment:
Are we not showing the exception in the front-end anymore?

Reply:
Restored in https://github.com/QuantStack/glue-lab/pull/75/commits/8c874a28ffbcf55dce248668a088ccd6f77bab31